package.json

```diff
@@ -47,18 +47,15 @@
   "devDependencies": {
     "babel-eslint": "^6.1.2",
     "eslint": "^3.1.1",
-    "jest-cli": "^12.0.0",
     "jsdoc": "^3.4.0",
+    "jest": "^15.1.1",
     "mkdirp": "^0.5.1"
   },
   "jest": {
     "testPathDirs": [
       "src",
       "bin",
       "sample"
-    ],
-    "unmockedModulePathPatterns": [
```

Comment:
automock was already manually turned off on every single file so this didn't do anything :)